client/src/scripts/managers/inputManager.ts

```diff
@@ -342,6 +343,50 @@ export class InputManager {
                 shootOnRelease = false;
             });
         }
+        const ticker = new PIXI.Ticker();
+        ticker.stop();
+        ticker.add(() => {
+            const gamepads = navigator.getGamepads();
+            if (gamepads[0]) {
```

Comment:
allow players to select their gamepad

Reply:
Here's an example of what it looks like:
![image](https://github.com/user-attachments/assets/69d229dc-79e2-4954-b1f9-d4626d29bb06)

Reply:
Also, here are what all of the different controller icons look like:
![image](https://github.com/user-attachments/assets/9f379c4f-fdb0-46a4-82fc-c3dd958dbaf6)